src/main/java/io/confluent/connect/elasticsearch/RetryUtil.java

```diff
@@ -160,13 +160,14 @@ public static long computeRetryWaitTimeInMillis(int retryAttempts,
         );
         return function.call();
       } catch (IOException e) {
-        log.warn("Failed to {} due to {}. Retrying attempt ({}/{})", description, e.getCause(), attempt, maxAttempts);
         if (attempt >= maxAttempts) {
+          log.error("Failed to {} due to {} after total of {} attempt(s)", description, e.getCause(), maxAttempts, e);
```

Comment:
let's do use `e.getMessage()` instead of `e` if you agree that `e` would results in printing the stack trace twice. To me it seems that the stack trace will be printed as part of `ConnectException`.